roles/network_plugin/cilium/templates/hubble/config.yml.j2

```diff
@@ -1,23 +1,23 @@
 ---
-# Source: cilium/templates/hubble-relay-configmap.yaml
+# Source: cilium helm chart: cilium/templates/hubble-relay/configmap.yaml
 apiVersion: v1
 kind: ConfigMap
 metadata:
   name: hubble-relay-config
   namespace: kube-system
 data:
   config.yaml: |
-    peer-service: unix:///var/run/cilium/hubble.sock
+    peer-service: "hubble-peer.kube-system.svc.cluster.local:443"
     listen-address: :4245
-    dial-timeout:
-    retry-timeout:
-    sort-buffer-len-max:
-    sort-buffer-drain-timeout:
+    metrics-listen-address: ":9966"
+    dial-timeout: 
+    retry-timeout: 
+    sort-buffer-len-max: 
+    sort-buffer-drain-timeout: 
     tls-client-cert-file: /var/lib/hubble-relay/tls/client.crt
     tls-client-key-file: /var/lib/hubble-relay/tls/client.key
     tls-hubble-server-ca-files: /var/lib/hubble-relay/tls/hubble-server-ca.crt
-    disable-server-tls: {% if cilium_hubble_tls_generate %}false{% else %}true{% endif %}
```

Comment:
Why do you remove TLS feature?

Reply:
It is generated by CronJob, right?

Reply:
I created the PR to fix that
https://github.com/kubernetes-sigs/kubespray/pull/9880

Reply:
@prashantchitta Could you take a look at #9880 of @utam0k ?
If that is good, we can keep a swtich of `disable-server-tls` based on the pull request.

Reply:
Done

Reply:
@oomichi @utam0k can you take a look now?